README.md

```diff
@@ -892,7 +918,7 @@ the `gpu-feature-discovery` component in standalone mode.
 
 The most basic installation command without any options is then:
 
-```
+```sh
```

Comment:
we use `shell` elsewhere in this document, could we use that here instead?

Reply:
Sure